meta-lmp-base/recipes-samples/images/lmp-feature-docker.inc

```diff
@@ -3,7 +3,7 @@ CORE_IMAGE_BASE_INSTALL += " \
     docker \
     docker-credential-helper-fio \
     python3-docker \
-    python3-docker-compose \
+    docker-compose \
```

Comment:
i think ricardo will want this done in a separate patch. that just slipped in during my debugging.

Reply:
ok. So, we need both `python3-docker-compose` and `docker-compose` in this PR?

Reply:
I think we'll switch to just "docker-compose" eventually (during this release). Not sure if we want to make that switch *now*?

Reply:
Yeah, please drop this one from this PR. We can do the switch in another pr.

Reply:
If I drop change to this file then just `python3-docker-compose` is installed. Then it's not clear how installation of the golang `docker-compose` that is patched in this PR is triggered at all.
So, I am asking whether we need both `python3-docker-compose` and `docker-compose` included?

Reply:
this PR is for building a correct docker-compose in factories that explicitly enable it. people can then build this in their factory if they really want it. then in another PR or commit (probably a PR) we do the switch from python to golang.

Reply:
updated